### PR TITLE
mingw-w64-cfitsio: Enable large file support for cfitsio

### PR DESCRIPTION
**Pull request by lock042**
It would be great to compile cfitsio with

-D_LARGEFILE_SOURCE \
-D_FILE_OFFSET_BITS=64 \

if not, there is a limitation of FITS size.

### CODE REVIEW

**Comment by Biswa96** [comment]
Increase `pkgrel` by 1.

**Comment by lock042** [comment]
Am I doing something wrong? CI is failing.

**Comment by Biswa96** [comment]
Increase `pkgrel` by 1. The CMake option should be `-D_LARGEFILE_SOURCE=ON`.

**Comment by lock042** [comment]
@Biswa96 : Thanks. Did it.

**Comment by Biswa96** [comment]
Also if possible, change the commit message. For example, "[the package you changed]: [what you have changed]". See other pull requests for the idea. Adding a commit message would also be helpful too.

**Comment by lock042** [comment]
@Biswa96 : I've changed the title of the MR, but don't think I can change my commit messages.
Sorry.

**Comment by lock042** [comment]
@Biswa96 : Is it ok for you?

**Comment by Biswa96** [comment]
` ¯\_(ツ)_/¯`

File: mingw-w64-cfitsio/PKGBUILD [hunk 1]
```diff
@@ -39,6 +39,8 @@ build() {
     -G"MSYS Makefiles" \
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
     -DBUILD_SHARED_LIBS=OFF \
+    -D_LARGEFILE_SOURCE=ON \
+    -D_FILE_OFFSET_BITS=64 \
```
Comment (1480c1) [inline mingw-w64-cfitsio/PKGBUILD:43]: I'm not sure if these have any effect as none of these are present or used in the CMakeLists

if you want these to have any effect, you might want to add
```bash
CFLAGS+=" -D_LARGEFILE_SOURCE=1 -D_FILE_OFFSET_BITS=64" \
    ${MINGW_PREFIX}/bin/cmake.exe \
```

Comment (lock042) [reply mingw-w64-cfitsio/PKGBUILD:44]: Oh thanks @1480c1 .
I'm really not a cmake user/dev

**Comment by lock042** [comment]
I hope this one is ok :).

File: mingw-w64-cfitsio/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-cfitsio/PKGBUILD:44]: I think @1480c1 meant that you should remove the cmake arguments in favor of CFLAGS.

Comment (lock042) [reply mingw-w64-cfitsio/PKGBUILD]: ok. Sorry. Done

**Pull request merged**